cider/exec/plan/parser/PlanRelVisitor.cpp

```diff
@@ -431,15 +433,15 @@ void ReadRelVisitor::visit(InputDescContext* input_desc_context) {
   for (int i = 0; i < rel_node_.base_schema().struct_().types_size(); i++) {
     auto type = rel_node_.base_schema().struct_().types(i);
     auto sti = getSQLTypeInfo(type);
-    ColumnInfoPtr colum_info_ptr =
-        std::make_shared<ColumnInfo>(100, cur_table_id, i, "col_" + i, sti, false);
+    ColumnInfoPtr colum_info_ptr = std::make_shared<ColumnInfo>(
+        fake_db_id, cur_table_id, i, fmt::format("col_{}", i), sti, false);
```

Comment:
When will `"col_" + i` not work and how does it looks like?

Reply:
`"col_" + i` means `"col_"[i]`

Reply:
Verified. Yes we should wrap this `i` with `std::to_string.